### DISCUSSION TITLE
[Feature Request] System Monitor in Taskbar

### DISCUSSION THREAD

**Discussion by kissandfly** [ramensoftware/windhawk-mods#1118] [Mod ideas]
Would be great to have like image attached, CPU GPU % usage and temps
![Capture d'écran 2024-10-21 095243](https://github.com/user-attachments/assets/83c8d7bd-939a-409a-9875-79ab528cadc6)

**Comment by yalov** [comment]
another example [taskbar-stats](https://openhoangnc.github.io/taskbar-stats/) (win11 does not supported, therefore a windhawk mod is appreciated)

![image](https://github.com/user-attachments/assets/36a34fec-3cd3-4fe4-8fda-6939c6f82fe8)

**Reply by Anixx** [reply]
> another example [taskbar-stats](https://openhoangnc.github.io/taskbar-stats/) (win11 does not supported, therefore a windhawk mod is appreciated)
> 
> ![image](https://private-user-images.githubusercontent.com/12796450/400904945-36a34fec-3cd3-4fe4-8fda-6939c6f82fe8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc4NDE2NzUsIm5iZiI6MTczNzg0MTM3NSwicGF0aCI6Ii8xMjc5NjQ1MC80MDA5MDQ5NDUtMzZhMzRmZWMtM2NkMy00ZmU0LThmZGEtNjkzOWM2ZjgyZmU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTI1VDIxNDI1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJiMzlhNmQwOGIxYTVkN2M0YWRiYmZjYmVkZDhiOTViZWRlMTg2OGNmYzZjOGRlMjUyOTg2OWFmMmU0MjU2Y2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.7qKHIm24HeCBB3C1_XfvEvdm4aoRj_w3N003Blvs49M)

Use Win10 taskbar on Win11. I think, this should work with EP taskbar as well.

**Comment by yalov** [comment]
I have tried to use generative AIs to make windhawk plugin, 
just for putting "Demo Text" on the left of the system tray icons on win11 — no luck.
Any help?
